test/unit/generator/content/schedule/part3/get-reduction-summary.test.js

```diff
@@ -0,0 +1,36 @@
+const getReductionSummary = require('../../../../../../app/generator/content/schedule/part3/get-reduction-summary')
+const { reductionSchedule } = require('../../../../../mocks/mock-schedule')
+const remainingAmount = require('../../../../../mocks/components/remaining-amount')
+
+describe('Get reduction summary', () => {
+  test('should return "The table below shows your revised quarterly payments.\n\n" for key stack 1 entry', () => {
+    const result = getReductionSummary(reductionSchedule)
+    expect(result.stack[0]).toBe('The table below shows your revised quarterly payments.\n\n')
+  })
+
+  test('should use solid line style for grid', () => {
+    const result = getReductionSummary(reductionSchedule)
+    expect(result.stack[1].layout.hLineStyle()).toBe('solid')
+    expect(result.stack[1].layout.vLineStyle()).toBe('solid')
+  })
+
+  test('should return "Current agreement value: £1,000.00\n" for key stack 2 line 1` entry', () => {

```

Comment:
These 3 tests below need changing as well to check that it is `schedule.adjustment.currentValue`, `schedule.adjustment.newValue` and `schedule.adjustment.adjustmentValue`

Which will reveal a problem w/ our mock schedule object which we need to fix...

Reply:
Super, would update the test name to reflect these changes too